src/vivarium/framework/engine.py

```diff
@@ -231,26 +231,26 @@ def initialize_simulants(self) -> None:
         self._clock.step_backward()
         population_size = pop_params.population_size
         self.simulant_creator(population_size, {"sim_state": "setup"})
-        self._clock.step_forward(self._population.get_population(True).index)
+        self._clock.step_forward(self.get_population().index)
```

Comment:
I did a refactor here, since sim has a `get_population()` method, and it seemed weird not to use it.

Reply:
is the `get_population` method something you added somewhere along the way as part of this variable time clock work? If not, are you very sure you understand why this wasn't already using it?

Reply:
It is fairly old code. What seems to have happened is the original method had some hard-coded lifecycle restrictions, which were superceded by the formalization of lifecycle restrictions and so removed.
https://github.com/ihmeuw/vivarium/pull/107/files
https://github.com/ihmeuw/vivarium/pull/109/commits/62311cc97469b6398d4c60f050a9dca404c6477e
Since `self.get_population()` is now little more than a wrapper for `self._population.get_population(True)` I didn't see a strong reason not to use it.